packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -47,32 +46,36 @@ contract BridgePool is Testable {
         address l1Token;
         address slowRelayer;
     }
-    struct Deposit {
-        DepositState depositState;
-        DepositType depositType;
+
+    // A Relay is linked to a Deposit object originating from L2
+    struct Relay {
+        RelayState relayState;
+        RelayType relayType;
         // A deposit can have both a slow and an instant relayer if a slow relay is "sped up" from slow to instant.
         // We want to store both addresses for separate payouts.
         address instantRelayer;
-        // @dev: See note above about why some Deposit params are collapsed into `RelayAncillaryDataContents`.
-        RelayAncillaryDataContents relayData;
     }
-    // Associates each deposit with a unique ID.
-    mapping(uint64 => Deposit) public deposits;
-    // If a deposit is disputed, it is removed from the `deposits` mapping and added to the `disputedDeposits` mapping.
-    // There can only be one disputed deposit per relayer for each deposit ID.
-    // @dev The mapping is `depositId-->disputer-->Deposit`
-    mapping(uint64 => mapping(address => Deposit)) public disputedDeposits;
+    // Associates each relay with a unique ancillary data hash derived from its constituent data. We need to key by the
+    // ancillary data so that the OptimisticOracle can locate relays on callbacks using only price requests' ancillary
+    // data
+    mapping(bytes32 => Relay) public relays;
+    // If a relay is disputed, it is removed from the `relays` mapping and added to the `disputedRelays` mapping.
+    // There can only be one disputed relay per relayer for each relay ancillary data.
```

Comment:
This comment is a little confusing -- here it says that the relayer is unique but the comment on the next line seems to indicate that the disputer is unique.

Reply:
Ah yes I actually change this mapping to be `mapping(bytes32 => Relay) disputedRelays` in a follow up PR #3281 so I'm going to ignore this comment for now.

Reply:
Sounds good!